rups/com/lowagie/rups/model/XfaFile.java

```diff
@@ -53,6 +53,8 @@ public XfaFile(OutputStreamResource resource) throws IOException, DocumentExcept
 		resource.writeTo(baos);
 		ByteArrayInputStream bais = new ByteArrayInputStream(baos.toByteArray());
 		SAXReader reader = new SAXReader();
+		// Note: No XML parsing security is configured here. May be done in the future
```

Comment:
wonder if we should throw a not imple,ented exception to be sure?

Reply:
The RUPS sub project is a separate tool from the iText core library. It is not built at all and not included in the iText main JAR that the build produces.